modules/vaos/spec/services/appointment_requests_service_spec.rb

```diff
@@ -60,11 +59,37 @@
     let(:start_date) { Date.parse('2019-08-20') }
     let(:end_date) { Date.parse('2020-08-22') }
 
-    context 'without data params' do
-      it 'returns an array of size 40' do
+    context 'without data params but with cc appointment data' do
+      it 'includes the cc appointment data' do
         VCR.use_cassette('vaos/appointment_requests/get_requests', match_requests_on: %i[method uri]) do
           response = subject.get_requests
-          expect(response[:data].size).to eq(40)
+          request_with_cc = response[:data][40]
+          expect(request_with_cc.cc_appointment_request).to eq(
+            data_identifier: {},
+            patient_identifier: {},
+            surrogate_identifier: {},
```

Comment:
we might need to drop some of these parameters if they include PII. I know patient_identifier does this a lot. You should be able to find some examples where I've needed to normalize the data for the FE.

Reply:
i *think* this PR will address this if i'm not mistaken.